classifiers/reference_quality/special_character_classifier/__init__.py

```diff
@@ -0,0 +1,37 @@
+import unicodedata
+from typing import Optional, List, Tuple
+from pydantic import BaseModel
+
+INPUT_EXAMPLE = {
+    "text": "Super funny haha 😀.",
+    "allowedRanges": None
+}
+
+ALLOWED_RANGES = set(range(32, 127)).union( # Basic Latin
+    set(range(160, 255)), # Latin-1 Supplement
+    set(range(256, 384)),  # Latin Extended-A
+    set(range(384, 592)),  # Latin Extended-B
+    set(range(8192, 8303)),  # General Punctuation
+    set(range(8352, 8399)),  # Currency Symbols
+    set([ord("\t"), ord("\n"), ord("\r")])# common stop chars
+)
+
+class SpecialCharacterClassifierModel(BaseModel):
+    text: str
+    allowedRanges: Optional[List[int]] = None
```

Comment:
maybe the name of the variable might be misleading. allowedRange (without s)

Reply:
can also be ignored

Reply:
Implemented. 👍